### ISSUE TITLE
Query id

### ISSUE THREAD

**Issue by xndjzx** [Cristinooo/boinkers-bot#1]
Please add query id please

**Comment by Cristinooo** [comment]
The query ID is yours, not mine.You can find how to use it in the manual.

**Comment by xndjzx** [comment]
Yes. But how to input in the script?? There are no txt file or json for query id.

**Comment by xndjzx** [comment]
Cannot use. Stuck pydantic_core and module 'bot'. Cannot run and install the module

**Comment by Cristinooo** [comment]
![截屏2024-10-23 上午7 15 36](https://github.com/user-attachments/assets/54167577-fa9c-44d6-b3c2-3979c44ce742)
I can't upload this private file. You can create one yourself and fill in the content.
